src/CommonLib/Enums/TrustAttributes.cs

```diff
@@ -15,6 +15,8 @@ public enum TrustAttributes
         TrustUsesRc4 = 0x80,
         TrustUsesAes = 0x100,
         CrossOrganizationNoTGTDelegation = 0x200,
-        PIMTrust = 0x400
+        PIMTrust = 0x400,
+        // We believe this deprecated flag represents a parent/child relationship
+        DeprecatedParentChild = 0x400000,
```

Comment:
Since there's no documentation to be found on this flag, this is the name I gave it but feel free to suggest something different.